### PR TITLE
docs: added release apache apisix 2.7 English version blog

### PR DESCRIPTION
**Pull request by Serendipity96**
Changes:

docs: added release apache apisix 2.7 English version blog

### CODE REVIEW

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+In version 2.7, we have developed new features of TCP proxy, including:
+
+- Allow domain name configuration in upstream
+- Allow mqtt-proxy plugin to configure domain name
+- Support receiving TLS over TCP connection, the certificate can be dynamically configured like HTTPS certificate
+- Route rules  based on SNI
+- Verify client certificate dynamically
+
+In subsequent versions, we will continue to  improve TCP proxy,  stay tuned!
```
Comment (juzhiyuan) [inline website/blog/2021-06-29-release-apache-apisix-2.7-en.md]:
```suggestion
In subsequent versions, we will continue to improve TCP proxy,  stay tuned!
```

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+- gzip
+- real_ip
+- proxy_max_temp_file_size
+
+## Download
+
+Download Apache APISIX 2.7.0-Release
+
+- Source code: please visit [download page](https://apisix.apache.org/downloads/)
+- Binary installation package: please visit [installation guide](https://apisix.apache.org/zh/docs/apisix/how-to-build/)
```
Comment (juzhiyuan) [inline website/blog/2021-06-29-release-apache-apisix-2.7-en.md]:
```suggestion
- Binary installation package: please visit [Installation Guide](https://apisix.apache.org/zh/docs/apisix/how-to-build/)
```

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Apache APISIX 2.7.0 has been released! Welcome to download and use it.
+
+This version supports multi-language plugins, enhances the four-layer TCP proxy, and enhances Nginx configuration. More than 20 developers participated, and more than 70 pull requests have been submitted. The following is an introduction to the key features.
+
+## Release Notes
+
+### Multi-language plugin
+
+With the release of the first version of apisix-java-plugin-runner,  and apisix-go-plugin-runner completes main function,  the multi-language plugin of Apache APISIX supports the two most widely used back-end programming languages. If you are still worried that the development of Apache APISIX plugin will be limited to Lua ecosystem, please try our multi-language plugin runner to develop Java / Go plugins.
```
Comment (spacewander) [inline website/blog/2021-06-29-release-apache-apisix-2.7-en.md]:
```suggestion
With the release of the first version of apisix-java-plugin-runner, and apisix-go-plugin-runner completes its main functions, the multi-language plugin of Apache APISIX supports the two most widely used back-end programming languages. If you are still worried that the development of Apache APISIX plugin will be limited to Lua ecosystem, please try our multi-language plugin runner to develop Java / Go plugins.
```

Comment (iamayushdas) [reply website/blog/2021-06-29-release-apache-apisix-2.7-en.md]:
```suggestion
With the release of the first version of apisix-java-plugin-runner,  and apisix-go-plugin-runner completes main function, the multi-language plugin of Apache APISIX supports the two most widely used back-end programming languages. If you are still think that the development of Apache APISIX plugin will be limited to Lua ecosystem, please try our multi-language plugin runner to develop Java / Go plugins.
```

Comment (iamayushdas) [reply website/blog/2021-06-29-release-apache-apisix-2.7-en.md]: worried seems to be an out of context word here IMO

**Review by iamayushdas — COMMENTED** [review]
other LGTM, thank you for this

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md (hunk 3 above)
Comment (spacewander) [reply website/blog/2021-06-29-release-apache-apisix-2.7-en.md]: `worried` is fine... The original Chinese text uses the same word.

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+authorURL: "https://github.com/Serendipity96"
+authorImageURL: "https://avatars.githubusercontent.com/u/23514812?v=4"
+---
+> [@Serendipity96](https://github.com/Serendipity96), Apache APISIX contributor from [Shenzhen Zhiliu Technology Co.](https://www.apiseven.com/)
+>
+<!--truncate-->
+
+Apache APISIX 2.7.0 has been released! Welcome to download and use it.
+
+This version supports multi-language plugins, enhances the four-layer TCP proxy, and enhances Nginx configuration. More than 20 developers participated, and more than 70 pull requests have been submitted. The following is an introduction to the key features.
```
Comment (Yiyiyimu) [inline website/blog/2021-06-29-release-apache-apisix-2.7-en.md]:
```suggestion
This version supports multi-language plugins, enhances the four-layer TCP proxy and Nginx configuration. More than 20 developers participated, and 70 plus pull requests have been submitted. The following is an introduction to the key features.
```

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md (hunk 3 above)
Comment (Yiyiyimu) [reply website/blog/2021-06-29-release-apache-apisix-2.7-en.md]:
```suggestion
With the first release of apisix-java-plugin-runner, and the completion of apisix-go-plugin-runner's main functions, the multi-language plugin of Apache APISIX now supports the two most widely used back-end programming languages. If you are still worried that the plugin development of Apache APISIX will be limited to Lua ecosystem, welcome to use our multi-language plugin runner to develop your own Java / Go plugins!
```

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+With the release of the first version of apisix-java-plugin-runner,  and apisix-go-plugin-runner completes main function,  the multi-language plugin of Apache APISIX supports the two most widely used back-end programming languages. If you are still worried that the development of Apache APISIX plugin will be limited to Lua ecosystem, please try our multi-language plugin runner to develop Java / Go plugins.
+
+### Enhanced four-layer TCP proxy
+
+In version 2.7, we have developed new features of TCP proxy, including:
+
+- Allow domain name configuration in upstream
+- Allow mqtt-proxy plugin to configure domain name
+- Support receiving TLS over TCP connection, the certificate can be dynamically configured like HTTPS certificate
+- Route rules  based on SNI
```
Comment (Yiyiyimu) [inline website/blog/2021-06-29-release-apache-apisix-2.7-en.md]:
```suggestion
- SNI-based route rules
```

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+With the release of the first version of apisix-java-plugin-runner,  and apisix-go-plugin-runner completes main function,  the multi-language plugin of Apache APISIX supports the two most widely used back-end programming languages. If you are still worried that the development of Apache APISIX plugin will be limited to Lua ecosystem, please try our multi-language plugin runner to develop Java / Go plugins.
+
+### Enhanced four-layer TCP proxy
+
+In version 2.7, we have developed new features of TCP proxy, including:
+
+- Allow domain name configuration in upstream
+- Allow mqtt-proxy plugin to configure domain name
+- Support receiving TLS over TCP connection, the certificate can be dynamically configured like HTTPS certificate
```
Comment (Yiyiyimu) [inline website/blog/2021-06-29-release-apache-apisix-2.7-en.md]:
```suggestion
- Support for receiving TLS over TCP connections, the certificate of which can be dynamically configured just like HTTPS certificate
```

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+- Route rules  based on SNI
+- Verify client certificate dynamically
```
Comment (Yiyiyimu) [inline website/blog/2021-06-29-release-apache-apisix-2.7-en.md]:
```suggestion
- Dynamic verification of client certificates
```

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+In subsequent versions, we will continue to improve TCP proxy,  stay tuned!
```
Comment (Yiyiyimu) [inline website/blog/2021-06-29-release-apache-apisix-2.7-en.md]:
```suggestion
In future releases, we will continue to improve TCP proxy, so stay tuned!
```

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+### Enhanced Nginx configuration
+
+We hope to set more and more Nginx configurations dynamically, so we add our own patches and Nginx C modules to enhance native Nginx.
```
Comment (Yiyiyimu) [inline website/blog/2021-06-29-release-apache-apisix-2.7-en.md]:
```suggestion
We hope to dynamically set more and more Nginx configurations, so we add our own patches and Nginx C modules to enhance the native Nginx.
```

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+
+The following new features are currently included:
+
+- Set mTLS dynamically
+- Set client_max_body_size dynamically
```
Comment (Yiyiyimu) [inline website/blog/2021-06-29-release-apache-apisix-2.7-en.md]:
```suggestion
- Dynamically set mTLS 
- Dynamically set client_max_body_size
```

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+
+In subsequent versions, we will continue to set the following Nginx configurations to dynamically:
```
Comment (Yiyiyimu) [inline website/blog/2021-06-29-release-apache-apisix-2.7-en.md]:
```suggestion
In future releases, we will continue to allow the following Nginx configurations to be set dynamically:
```

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md (hunk 3 above)
Comment (1502shivam-singh) [reply website/blog/2021-06-29-release-apache-apisix-2.7-en.md]: Agree with @iamayushdas here. I guess "hesitant" should be the right word choice here, regardless of language @spacewander as in "If you are still **hesitant** that the plugin development of Apache APISIX will be limited to Lua ecosystem". What do you think?

Comment (1502shivam-singh) [inline website/blog/2021-06-29-release-apache-apisix-2.7-en.md]:
```suggestion
With the release of the first version of apisix-java-plugin-runner,  and apisix-go-plugin-runner completes main function,  the multi-language plugin of Apache APISIX supports the two most widely used back-end programming languages. If you are still hesitant that the development of Apache APISIX plugin will be limited to Lua ecosystem, please try our multi-language plugin runner to develop Java / Go plugins.
```

Comment (iamayushdas) [reply website/blog/2021-06-29-release-apache-apisix-2.7-en.md]: Exactly , this is what i was saying

Comment (spacewander) [reply website/blog/2021-06-29-release-apache-apisix-2.7-en.md]: Using "hesitant" is a good choice. But I think being strict with the original text is better.

Comment (spacewander) [reply website/blog/2021-06-29-release-apache-apisix-2.7-en.md]: I don't want to waste time arguing a word. This is no English course. As for translation, we should not have so much secondary creation.

Comment (1502shivam-singh) [reply website/blog/2021-06-29-release-apache-apisix-2.7-en.md]: Oh, Makes sense 👍

File: website/blog/2021-06-29-release-apache-apisix-2.7-en.md (hunk 4 above)
Comment (Serendipity96) [reply website/blog/2021-06-29-release-apache-apisix-2.7-en.md]: "plus" is better than "more than"?

Comment (Yiyiyimu) [reply website/blog/2021-06-29-release-apache-apisix-2.7-en.md]: it's a bit duplicated: `"More than" 20 developers participated, and "more than" 70 pull requests have been submitted`

I think changing it to another expression could be fluent

**Review by tokers — APPROVED** [review]

**Review by moonming — APPROVED** [review]

**Pull request merged**